sql/_insert_returning.sql

```diff
@@ -0,0 +1 @@
+INSERT INTO %s (%s) VALUES (%s) RETURNING *
```

Comment:
```suggestion
INSERT INTO %s (%s) VALUES (%s) RETURNING *

```

Reply:
I did it on purpose. The current queries don't have it. Should I add it for them too?

Reply:
Not really sure. This is more about the linter. I just saw Github was complaining about it and... 🤔